src/main/java/UserDetails.java

```diff
@@ -0,0 +1,88 @@
+import java.util.Scanner;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+
```

Comment:
A very interesting feature, maybe try to breakuser information down to seperate classes in order to make it more Object-Oriented